demoScripts/aksQuickstartEventGrid.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Subscribe to Azure Kubernetes Service events with Azure Event Grid
+description: Use Azure Event Grid to subscribe to Azure Kubernetes Service events
+services: container-service
+author: zr-msft
+ms.topic: article
+ms.date: 07/12/2021
+ms.author: zarhoads
+---
+
+# Quickstart: Subscribe to Azure Kubernetes Service (AKS) events with Azure Event Grid
+
+Azure Event Grid is a fully managed event routing service that provides uniform event consumption using a publish-subscribe model.
+
+In this quickstart, you'll create an AKS cluster and subscribe to AKS events.
+
+## Prerequisites
+
+* An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
+* [Azure CLI][azure-cli-install] or [Azure PowerShell][azure-powershell-install] installed.
+
+## Create an AKS cluster
+
+### [Azure CLI](#tab/azure-cli)
+
+## Define Environment Variables
+```azurecli-interactive
+export RESOURCE_GROUP_NAME=myResourceGroup
+export RESOURCE_LOCATION=eastus
+export AKS_CLUSTER_NAME=myAKSCluster
+export NAMESPACE_NAME="myNamespace$(printf "%08d" $((RANDOM%100000000)))"
+export EVENT_GRID_HUB_NAME=myEventGridHub
+export EVENT_GRID_SUBSCRIPTION_NAME=myEventGridSubscription
+```
+
+Create an AKS cluster using the [az aks create][az-aks-create] command. The following example creates a resource group *MyResourceGroup* and a cluster named *MyAKS* with one node in the *MyResourceGroup* resource group:
+
+```azurecli-interactive
+az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION
+az aks create --resource-group $RESOURCE_GROUP_NAME --name $AKS_CLUSTER_NAME --location $RESOURCE_LOCATION  --node-count 1 --generate-ssh-keys
+```
+
+### [Azure PowerShell](#tab/azure-powershell)
```

Comment:
Not an Innovation Engine problem, but we should not have PowerShell and Bash intertwined like this. We should take this up with Docs team. Tabs I can cope with, as long as BASH is the default, but not inline merging like this.